README.md

```diff
@@ -42,6 +42,7 @@ prettierx <options> <file(s)>
 | Align ternary lines                                  | `true`        | `--no-align-ternary-lines`                 | `alignTernaryLines: <bool>`                | Align ternary lines in case of multiline ternery term (default behavior, which is in conflict with ESLint/StandardJS behavior). Should be disabled for consistency with ESLint/StandardJS behavior. |
 | paren spacing                                        | `false`       | `--paren-spacing`                          | `parenSpacing: <bool>`                     | Print spaces between parens, WordPress style (not recommended with default `arrowParens: "always" setting).                                                                                         |
 | break before else                                    | `false`       | `--break-before-else`                      | `breakBeforeElse: <bool>`                  | Always add a line break before else.                                                                                                                                                                |
+| break before statement                               | `never`       | `--break-before-statement`                 | see [./docs/options.md](./docs/options.md) | Always add a line break before statement following if, else, which, for, for await, for each, do, and while.                                                                                        |
```

Comment:
```suggestion
| break before statement                               | `never`       | `--break-before-statement`                 | see [./docs/options.md](./docs/options.md) | Always add a line break before statement following if, else, for, for await, for each, do, while, and with.                                                                                        |
```

TODO: fix spacing to avoid lint issue

rationale:

- replace "which" which was incorrect
- I would rather keep "with" at the end, since it seems to be almost never used and not recommended ref: https://2ality.com/2011/06/with-statement.html